resources/apt/apt.dsc.resource.json

```diff
@@ -0,0 +1,73 @@
+{
+    "$schema": "https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2024/04/bundled/resource/manifest.json",
+    "type": "DSC.PackageManagement/Apt",
+    "description": "DSC resource to manage Apt packages",
+    "tags": [
+        "Linux",
+        "apt",
+        "PackageManagement"
+    ],
+    "version": "0.1.0",
+    "get": {
+        "executable": "apt.dsc.resource.sh",
+        "args": [
+            "get"
+        ],
+        "input": "env"
+    },
+    "set": {
+        "executable": "apt.dsc.resource.sh",
+        "args": [
+            "set"
+        ],
+        "input": "env",
+        "implementsPretest": true,
```

Comment:
Does it handle pretest? These lines look to me like set will just call `apt install` or `apt remove` - when this property is set to `true`, DSC assumes that the resource is testing whether to run code that changes the system before doing so.

```sh
    if [[ $_exist = true ]]; then
        apt install -y "${packageName}"
    else
        apt remove -y "${packageName}"
    fi
```